src/main/java/com/eppo/sdk/helpers/Shard.java

```diff
@@ -42,9 +42,9 @@ public static String getHex(String input) {
      * @return
      */
     public static int getShard(String input, int maxShardValue) {
-        String hashText = Shard.getHex(input);
+        StringBuilder hashText = new StringBuilder(Shard.getHex(input));
         while (hashText.length() < 32) {
-            hashText = "0" + hashText;
+            hashText.insert(0, "0");
```

Comment:
👍 if you're going to write Java, do it the Java way 😄

Reply:
Ha yeah. In the former way, the compiler would probably just optimize to this anyways. But doing this appeased the suggestion from WebStorm 😅